webpack.global.config.js

```diff
@@ -45,13 +46,36 @@ module.exports = {
         },
       ],
     }),
+    new RemovePlugin({
+      after: {
+        root: './dist',
+        include: [
+          'canopy/fonts/lyon'
+        ],
+      }
+    })
   ],
   mode: 'production',
   module: {
     rules: [
       {
         test: /\.scss$/,
-        use: [MiniCssExtractPlugin.loader, 'css-loader', 'sass-loader'],
+        use: [
+          { loader: MiniCssExtractPlugin.loader },
+          {
+            loader: 'css-loader',
+            options: {
+              url: (url) => {
+                // TODO: use regex for more precise targeting
```

Comment:
Have you done the TODO?

Reply:
Yes, I did it but forgot to remove the comment 🤦